cmd/main.go

```diff
@@ -470,3 +473,14 @@ func setupWebhooks(mgr ctrl.Manager, currentNamespace string, validateClusterUpg
 	}
 	return nil
 }
+
+type comaSeperated []string
```

Comment:
just wondering: why did you choose that approach instead of, for example, a plain implementation of the `flag.Value` interface? both are seem the same, the latter is more popular, i'd say (and more flexible)